File: 3sem/poo/Aula01-Codigo/C++/Exemplo04-Heranca/Aquaticos.cpp
#include <string>
#include "Animal.cpp"

using namespace std;

class Aquaticos : public Animal{
   private:
    int iMinsDeFolego;

   public:
    Aquaticos(string saNome, int umMinsDeFolego) : Animal(saNome){
        iMinsDeFolego = umMinsDeFolego;
    };

    int getMinsDeFolego() {
        return iMinsDeFolego;
    };

};

File: 3sem/poo/Aula01-Codigo/Java/Exemplo04-Heranca/src/exemplo4heranca/Aquaticos.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package exemplo4heranca;

/**
 *
 * @author Junio
 */
public class Aquaticos extends Animal{
    int iMinsDeFolego;

    public Aquaticos(String saNome, int umMinsDeFolego) {
        super(saNome);
        iMinsDeFolego = umMinsDeFolego;
    }

    public int getMinsDeFolego() {
        return iMinsDeFolego;
    }

}
